docs/api.rst

```diff
@@ -259,6 +268,65 @@ defaults in the future might change.
 Inside the templates the behaviour can be temporarily changed by using
 the `autoescape` block (see :ref:`autoescape-overrides`).
 
+Note that :func:`~jinja2.select_autoescape` offers also a parameter
+`special_extensions` that allows you to define a custom escape function,
+i.e. if you handling LaTeX files
+(see description of :func:`~jinja.select_autoescape` above for example).
+
+To write your own custom escape function simply return it for the
+wanted template extension. For instance if you are fan of peace in
```

Comment:
This sentence sounds a bit rough, but I can't come up with something better, as I am not sure what you intend to say.

As far as I see you do not return the custom function, but configure the Environment with it, right?